File: pyg_order_interface/src/main/java/com/pyg/order/service/PayLogService.java
package com.pyg.order.service;
import java.util.List;
import java.util.Set;

import com.pyg.pojo.TbPayLog;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface PayLogService {

	/* 
	 * @Desc: 分页查询
	 * @Date: 2018/12/15 
	 */
	public PageResult findPage(TbPayLog payLog, int pageNum, int pageSize,String createTime);
}

File: pyg_manage_web/src/main/webapp/js/service/payLogService.js
//服务层
app.service('payLogService',function($http){

	//搜索
	this.search=function(page,rows,searchEntity,createTime){
		return $http.post('../payLog/search.do?page='+page+"&rows="+rows+"&createTime="+createTime, searchEntity);
	};
});
